deployment/ansible/inventory/aws/group_vars/all.yaml

```diff
@@ -0,0 +1,23 @@
+chain_id: test-net2
+
+dcl_home: /var/lib/dcl/.dcl
+dcl_version: 0.9.0
+dcld:
+  version: "{{ dcl_version }}"
+  path: "{{ dcl_home }}/cosmovisor/genesis/bin/dcld"
+cosmovisor:
+  version: "{{ dcl_version }}"
+  user: cosmovisor
+  group: dcl
+  path: /usr/bin/cosmovisor
+  home: "{{ dcl_home | dirname }}"
+
+dcld_checksums:
+  0.9.0: c333d828a124e527dd7a9c0170f77d61ad07091d9f6cd61dd0175a36b55aadce
+cosmovisor_checksums:
+  0.9.0: c05705efe5369b9d83e65ef7b252bd7c610eec414ae3f6c08681bcf49dc38e6d
+
+dcld_download_url: "https://github.com/zigbee-alliance/distributed-compliance-ledger/releases/download/v{{ dcld.version }}/dcld"
+dcld_binary_checksum: "sha256:{{ dcld_checksums[dcld.version] }}"
```

Comment:
I don't think that you have to provide this information, given it has been defined as default values

Reply:
It is done to ensure consistency between all roles used in deploy.yml. Otherwise, they can have different values for a single deployment. According to the [ansible variables precedence rule](https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#understanding-variable-precedence), it should replace default values